### ISSUE TITLE
Compilation failure on android with webpack

### ISSUE THREAD

**Issue by mevinDhun** [abhayastudios/nativescript-contacts-lite#1]
Hi, i tried using your plugin with webpack but it fails on android.

To reproduce:
Create a sample app based on default angular template:
Add the webpack plugin + nativescript-contacts-lite + worker loader
Add some codes using the web worker

Run the bundled project
```
npm run start-android-bundle 
```
Gradle fails at :asbg:generateBindings
```
Warning: there already is an extend called com.tns.FragmentClass.
Warning: The static binding generator will generate extend from:vendor.js implementation
:asbg:generateBindings
Exception in thread "main" java.io.IOException: File already exists. This may lead to undesired behavior.
Please change the name of one of the extended classes.
File:/Users/mevindhunnooa/Documents/workspace/my-app-name/platforms/android/src/main/java/com/tns/FragmentClass.java Class: com.tns.FragmentClass
        at org.nativescript.staticbindinggenerator.Generator.writeBindings(Generator.java:68)
        at org.nativescript.staticbindinggenerator.Main.main(Main.java:15)
:asbg:generateBindings FAILED

FAILURE: Build failed with an exception.
```

According to  [#778](https://github.com/NativeScript/android-runtime/issues/778), This occurs because the plugin loads the tns modules which are already in the vendor.ts. I even tried using android@next version but it still fails. Here is a sample project with minimalistic code to reproduce [https://github.com/mevinDhun/nativescript-webpack-error-app](https://github.com/mevinDhun/nativescript-webpack-error-app)

Is there any way to fix this?

Thanks

**Comment by abhayastudios** [comment]
Hi Mevin,

I am traveling until next week, so I am less available to check things. However, I was able to run with webpack last time I checked. Thanks for the demo project, I will get bak to you soon.

Cheers!

**Comment by abhayastudios** [comment]
@mevinDhun I was able to reproduce with your sample project. However, since this seems to be an issue with the android runtime (see my addition to [this issue](https://github.com/NativeScript/android-runtime/issues/778), I don't think I have anything to change for now.

However, as a work around you can do the following:

- use the `getContacts` function instead of the `getContactsWorker`
- remove [these lines](https://github.com/abhayastudios/nativescript-contacts-lite/blob/master/index.js#L4-L36) from `index.js` in your `node_modules/nativescript-contacts-lite/index.js`

Then run `npm run start-android-bundle` and it will work. I will leave this issue open for now and keep an eye out to see when the issues with web workers will be fixed.

**Comment by mevinDhun** [comment]
@abhayastudios The ```getContacts``` will block the UI and is not suitable for my app. Using webpack is not a must for me right now, hence i can skip it for android. I will wait for [778](https://github.com/NativeScript/android-runtime/issues/778) to be resolved and then will try to integrate wepack on my android version. 
Thanks

**Comment by alexandruantonica** [comment]
@abhayastudios Did you find any solution for `getContactsWorker` to work on Android ?
According to [778](https://github.com/NativeScript/android-runtime/issues/778) the issues with web workers was solved.

**Comment by abhayastudios** [comment]
@alexandruantonica Indeed it is supposed to be released with NS 3.2. I haven't gotten around trying it yet in the @next build. Will update here when I get a chance.

**Comment by abhayastudios** [comment]
Fixed with NS3.2 and bb9f85a020ca718622cae3ca163c635b5043d7d5
